api/services/ocean/spark/schemas/oceanSparkCluster.yaml

```diff
@@ -0,0 +1,56 @@
+type: object
+required:
+  - id
+  - controllerClusterId
+  - oceanClusterId
+  - state
+  - region
+  - k8sVersion
+  - nodeCount
+  - runningAppCount
+  - oceanControllerLastHeartbeat
+  - operatorLastHeartbeat
+  - createdAt
+  - updatedAt
+allOf:
+- $ref: "../../../../commons/schemas/createdAtUpdatedAt.yaml"
+- type: object
+  description: >
+    The description of an Ocean Spark cluster
+  properties:
+    id:
+      $ref: "oceanSparkClusterId.yaml"
+    controllerClusterId:
+      $ref: "controllerClusterId.yaml"
+    oceanClusterId:
+      $ref: "oceanClusterId.yaml"
+    state:
+      $ref: "oceanSparkClusterState.yaml"
+    region:
+      type: string
+      example: "us-west-2"
+      description: >
+        The region where the Ocean Spark cluster lives
+    k8sVersion:
+      type: string
+      description: >
+        The Kubernetes API of the Ocean Spark cluster
+      example: "v1.21.2-eks-0389ca3"
+    nodeCount:
+      type: integer
+      description: The number of nodes currently provisioned in the Ocean Spark cluster
+      example: 1
+    runningAppCount:
+      type: integer
+      description: The number of Spark applications currently running in the Ocean Spark cluster
+      example: 3
+    oceanControllerLastHeartbeat:
+      type: string
+      format: date-time
+      description: The last heartbeat sent by the Ocean controller in the Ocean cluster
```

Comment:
nit: The last hearbeat "received"

Is even more specific (e.g. a networking issue may prevent us from receiving the hearbeats)

Same comment on operatorLastHeartbeat

Reply:
Thanks!